generators/entity-server/templates/src/main/java/package/repository/EntityRepository.java.ejs

```diff
@@ -38,11 +38,19 @@ import org.springframework.data.mongodb.repository.MongoRepository;
 import org.springframework.data.neo4j.repository.Neo4jRepository;
 <%_ } _%>
 <%_ if (databaseTypeCouchbase) { _%>
```

Comment:
Now that everything was moved to `N1qlCouchbaseRepository`, we don't need this anymore, except when `searchEngineCouchbase` enabled after merging my PR.